lib/webauthn/attestation_statement.rb

```diff
@@ -13,26 +13,26 @@ class FormatNotSupportedError < Error; end
     ATTESTATION_FORMAT_ANDROID_KEY = "android-key"
     ATTESTATION_FORMAT_TPM = "tpm"
 
-    def self.from(format, statement)
+    def self.from(format, statement, relying_party: WebAuthn.configuration.relying_party)
```

Comment:
[We talked about this IRL, but writing a comment for the record]

This extra keyword argument feels a little bit odd, but I can't think of a better alternative right now... :-)

Reply:
I like it - passing configuration to be used as an argument, by default being a copy of the global configuration. I'm a little torn on the 'relying party' name but I'll not 🚲🏡  too much here 😄

Reply:
Good to know :-)
Thanks.